fail2ban/server/server.py

```diff
@@ -324,6 +324,15 @@ def setUnbanIP(self, name, value):
 	def getBanTime(self, name):
 		return self.__jails[name].actions.getBanTime()
 	
+	def isAlive(self, jailnum=None):
+		if jailnum is not None and len(self.__jails) != jailnum:
+			return 0
+		for j in self.__jails:
+			j = self.__jails[j]
```

Comment:
doesn't look right -- j (call it "jail" please) should already be a jail, not an index... I guess this line is not necessary at all!  but also points to probably having no test for this portion of the code